examples/all-clusters-app/esp32/main/Kconfig.projbuild

```
@@ -101,6 +101,15 @@ menu "Demo"
         help
             To reduce wear and heat the M5Stack's Display is automatically switched off after a few seconds
 
+    #Each beard has a status led gpio, by using it we can dynamically set the pin number without judge the board type in the source code
```

```suggestion
    # Each board has a status led, define its pin number.
```
And move it to `help` section.

---

done